src/client/app/shared/string-utils.service.ts

```diff
@@ -38,7 +38,7 @@ export class StringUtils {
 			return null;
 		}
 
-		return s.replace(/\w\S*/g, (txt) => {
+		return s.replace(/\b\w\S*/g, (txt) => {
 			return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
```

Comment:
Looks like the regex matches one-char words, don't you need a null check on `txt.substr(1)` and a test case for the word "a"?